releases/reference-assemblies/README.md

```diff
@@ -46,8 +23,8 @@ Change `TargetFramework` property to `TargetFrameworks` and add a new .NET Frame
 Add Package reference:
 
 ```xml
-<ItemGroup Condition=" '$(TargetFramework)' == 'net472' ">
-  <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" />
+<ItemGroup>
```

Comment:
Why remove the Condition here? If the project is multitargetting, you would definitely want to keep the condition on.

Reply:
The way it works is that the targets within the package do the conditions, it was in a recommendation in another issue, I'll dig that out.

Reply:
Here: https://github.com/dotnet/designs/pull/33#issuecomment-489317138

Reply:
Yes, I think it's simpler to avoid the condition in the .csproj here.